modules/update-upgrading-oc-adm-upgrade-recommend.adoc

```diff
@@ -0,0 +1,94 @@
+// Module included in the following assemblies:
+//
+// * updating/updating_a_cluster/updating-cluster-cli.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="update-upgrading-oc-adm-upgrade-recommend_{context}"]
+= Finding recommended update paths with `oc adm upgrade recommend` (Technology Preview)
+
+When updating your cluster, it is useful to find out which versions are recommended for your update. While the `oc adm upgrade` command returns a list of the next available versions, this command decouples that information and narrows down your suggestions to recommend a new target release before you launch your update.
+
+The `oc adm upgrade recommend` command is read-only and will never alter any state in your cluster. 
+
+:FeatureName: The `oc adm upgrade recommend` command
+include::snippets/technology-preview.adoc[]
+
+[IMPORTANT]
+====
+While your cluster does not need to be a Technology Preview-enabled cluster, you must enable the `OC_ENABLE_CMD_UPGRADE_RECOMMEND` Technology Preview environment variable, otherwise the {oc-first} will not recognize the command and you will not be able to use the feature. 
+====
```

Comment:
Any way around stacking two admonitions here?

Reply:
after reading step 1: I would remove most of this and maybe just add it back as a note that a TP cluster isn't required, since you explicitly make them enable that flag in the proc anyway.